tests/components/ambient_network/snapshots/test_sensor.ambr

```diff
@@ -441,6 +449,57 @@
     'state': '2023-10-30T09:45:00+00:00',
   })
 # ---
+# name: test_sensors[AA:AA:AA:AA:AA:AA][sensor.station_a_last_update-entry]
```

Comment:
Please delete locally this file and generate it from scratch as it contains unused/obsolete entries

Reply:
Regenerated.